code/datums/martial/cqc.dm

```diff
@@ -225,7 +225,7 @@
 ///Subtype of CQC. Only used for the chef.
 /datum/martial_art/cqc/under_siege
 	name = "Close Quarters Cooking"
-	var/list/valid_areas = list(/area/service/kitchen)
+	var/list/valid_areas = list(/area/service)
```

Comment:
Just expand it to kitchen and bar. The chef shouldn't know Close Quarters Cooking when in the Janitor's little hideaway.

Reply:
What about botany? They're almost always next to the kitchen.

Reply:
> What about botany? They're almost always next to the kitchen.

t. chef main